Prac_07/KivyDemos-master/squaring.py

```diff
@@ -14,6 +14,7 @@
 
 class SquareNumberApp(App):
     """ SquareNumberApp is a Kivy App for squaring a number """
+
     def build(self):
         """ build the Kivy app from the kv file """
         Window.size = (200, 100)
```

Comment:
Upon opening the file, the window is quite squishy, try upping the window size a tad.